<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-FUGQNF4" ignored="3" name="Default test(failed)(failed)" tests="6" failures="1" timestamp="27 Jan 2023 02:57:33 GMT" time="0.063" errors="0">
  <testcase name="test3" time="0.015" classname="org.rerun.A">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase name="test3" time="0.0" classname="org.rerun.A">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase name="test3" time="0.0" classname="org.rerun.A">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase name="test3" time="0.016" classname="org.rerun.A">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase name="test3" time="0.0" classname="org.rerun.A">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase name="test3" time="0.0" classname="org.rerun.A">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.rerun.A.test3(A.java:21)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase name="test1" time="0.0" classname="org.rerun.A">
    <ignored/>
  </testcase> <!-- test1 -->
  <testcase name="test2" time="0.0" classname="org.rerun.A">
    <ignored/>
  </testcase> <!-- test2 -->
  <testcase name="test4" time="0.0" classname="org.rerun.A">
    <ignored/>
  </testcase> <!-- test4 -->
</testsuite> <!-- Default test(failed)(failed) -->
